### PR TITLE
Cleanup tests

### PR DESCRIPTION
**Pull request by iskandr**
* Deleted redundant transcript id tests
* Merged related test modules
* PEP8 fixes across many unit tests

[Review on Reviewable](https://reviewable.io/reviews/hammerlab/pyensembl/67)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -36,7 +36,7 @@ Ensembl data:
 pyensembl <list of Ensembl release numbers> install
 ```
 
-For example, `pyensembl 75 76 install` will download and install all
+For example, `pyensembl install --release 75 76` will download and install all
```
Comment (tavinathanson) [inline README.md]: Did you change the script somewhere and forgot to add it to the PR?

**Comment by tavinathanson** [comment]
Looks good minus comment!

File: README.md (hunk 1 above)
Comment (iskandr) [inline README.md]: I think Tim changed it in a previous PR, from shell.py:
```
To install particular release(s):
    %(prog)s install --release 75 77
```

Comment (tavinathanson) [inline README.md]: Ah gotcha

**Pull request merged**